book/09-git-and-other-scms/sections/client-bzr.asc

```diff
@@ -0,0 +1,102 @@
+==== Git and Bazaar
+
+There are many projects that allow you to use Git as a Bazaar client.
+Here we'll use Felipe Contreras' project that you may find at https://github.com/felipec/git-remote-bzr[].
+To install it, you just have to download the file git-remote-bzr in a folder contained in your `$PATH`:
+[source,console]
+----
+$ wget https://raw.github.com/felipec/git-remote-bzr/master/git-remote-bzr -O ~/bin/git-remote-bzr
+$ chmod +x ~/bin/git-remote-bzr
+----
+
+You also need to have Bazaar installed.
+
+That's all!
+
+===== Create a Git repository from a Bazaar repository
+
+It is simple to use.
+It is enough to clone a Bazaar repository prefixing it by `bzr::`.
+If you worked in a local Bazaar repository and pushed to a remote one, it is better to clone the remote.
+That will make your work simpler; else you must go through your local repository and you may have problems when pushing to the remote if your Bazaar and Git repositories were not up-to-date.
```

Comment:
Let's rephrase a bit:

> Since Git and Bazaar both do full clones to your machine, it's possible to attach a Git clone to your local Bazaar clone, but this isn't recommended. It's much easier to attach your Git clone directly to the same place your Bazaar clone is attached to - the central repository.

…or something like that. Does that make sense?